src/code/z_camera.c

```diff
@@ -3106,266 +6797,795 @@ void func_80057C6C(Camera* camera, View* view, CollisionContext* colCtx, GlobalC
+    s8* temp_t7;
+    s8* temp_t8;
+    s8* temp_t9;
+    Camera* temp_a0;
+    s32 phi_a1;
+    char* phi_v1;
+    char* phi_t0;
+    s32 phi_v0;
+    s32 phi_v0_2;
+
+    if (OREG(0) & 1 && camera->thisIdx == camera->globalCtx->activeCamera && !gDbgCamEnabled) {
+        for (phi_a1 = 0, phi_v1 = sp58, phi_t0 = sp48; phi_v1 < &sp58[4]; phi_v1++, phi_t0++, phi_a1++) {
+            temp_a0 = camera->globalCtx->cameraPtrs[phi_a1];
+            if (temp_a0 == NULL) {
+                *phi_v1 = '-';
+                *phi_t0 = ' ';
+            } else {
+                switch (temp_a0->status) {
+                    case 0:
+                        *phi_v1 = 'c';
+                        *phi_t0 = ' ';
+                        break;
+                    case 1:
+                        *phi_v1 = 'w';
+                        *phi_t0 = ' ';
+                        break;
+                    case 3:
+                        *phi_v1 = 's';
+                        *phi_t0 = ' ';
+                        break;
+                    case 7:
+                        *phi_v1 = 'a';
+                        *phi_t0 = ' ';
+                        break;
+                    case 0x100:
+                        *phi_v1 = 'd';
+                        *phi_t0 = ' ';
+                        break;
+                    default:
+                        *phi_v1 = '*';
+                        *phi_t0 = ' ';
+                        break;
+                }
+            }
+        }
+        *phi_v1++ = '\0';
+        *phi_t0++ = '\0';
+
+        sp48[camera->globalCtx->activeCamera] = 'a';
+        func_8006376C(3, 0x16, 5, sp58);
+        func_8006376C(3, 0x16, 1, sp48);
+        func_8006376C(3, 0x17, 5, "S:");
+        func_8006376C(5, 0x17, 4, sCameraSettingNames[camera->setting]);
+        func_8006376C(3, 0x18, 5, "M:");
+        func_8006376C(5, 0x18, 4, sCameraModeNames[camera->mode]);
+        func_8006376C(3, 0x19, 5, "F:");
+        func_8006376C(5, 0x19, 4,
+                      sCameraFunctionNames[sCameraSettings[camera->setting].cameraModes[camera->mode].funcIdx]);
+
+        phi_v0 = 0;
+        if (camera->camDataIdx < 0) {
+            sp50[phi_v0++] = '-';
+        }
+        phi_v0_2 = camera->camDataIdx / 0xA;
+        if (phi_v0_2 != 0) {
+            sp50[phi_v0++] = phi_v0 / 0xA + '0';
+        }
+        sp50[phi_v0++] = phi_v0 % 10 + '0';
+        sp50[phi_v0++] = ' ';
+        sp50[phi_v0++] = ' ';
+        sp50[phi_v0++] = ' ';
+        sp50[phi_v0++] = ' ';
+        sp50[phi_v0] = '\0';
+        func_8006376C(3, 26, 5, "I:");
+        func_8006376C(5, 26, 4, sp50);
+    }
 }
 #else
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/Camera_ChangeStatus.s")
+void Camera_PrintSettings(Camera* camera);
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/Camera_PrintSettings.s")
 #endif
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800584E8.s")
+s32 Camera_ChkWater(Camera* camera) {
+    f32 waterY;
```

Comment:
I know we use "chk" in things like ColChkInfo and stuff but I feel like in alot of these cases here the shortening isnt necessary. just in my imo tho FeelsOkayMan